judge/migrations/0145_problem_authorship.py

```diff
@@ -0,0 +1,60 @@
+        ),
```

Comment:
I know these are auto-added, but let's remove them from this migration file since they're not relevant to this PR. Ideally, we would have these changes in its only migration file.
```suggestion
```

Reply:
Done :)